clade/proto/sync.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
+
+package clade.sync;
+
+import "schema.proto";
+
+message DataSyncCommand {
+  // Path to the Delta Table, relative to the root URL of the store below.
+  // Will be created based on the schema of the RecordBatch if doesn't exist.
+  // e.g. path/to/table
+  string path = 1;
+
+  // Root URL (e.g. s3://my-bucket) and connection options for the table's path
+  schema.StorageLocation store = 2;
+
+  // Primary key columns. Must not be empty: the table must always
+  // have a replica identity / PK.
+  repeated string pk_column = 3;
```

Comment:
Should this be called `pk_columns`? (plural)

Reply:
Good point, it is indeed a [recommendation](https://protobuf.dev/programming-guides/style/#repeated-fields), revising.